### PR TITLE
Fix void teleport

### PR DESCRIPTION
**Pull request by xReveres**
Fixes problem with void teleport.

### CODE REVIEW

**Review by dlsf — APPROVED** [review]

File: src/main/java/com/iridium/iridiumskyblock/listeners/PlayerMoveListener.java [hunk 1]
```diff
@@ -50,7 +52,16 @@ public void onPlayerMove(PlayerMoveEvent event) {
 
         if (event.getTo().getY() < 0 & IridiumSkyblock.getInstance().getConfiguration().voidTeleport) {
             Optional<Island> island = IridiumSkyblock.getInstance().getIslandManager().getIslandViaLocation(event.getFrom());
-            island.ifPresent(value -> IridiumSkyblock.getInstance().getIslandManager().teleportHome(player, value, 0));
+            if (island.isPresent()) {
```
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumskyblock/listeners/PlayerMoveListener.java:55]: You will need to check we are in the Iridium's world

File: src/main/java/com/iridium/iridiumskyblock/listeners/PlayerMoveListener.java [hunk 2]
```diff
@@ -50,7 +52,16 @@ public void onPlayerMove(PlayerMoveEvent event) {
 
         if (event.getTo().getY() < 0 & IridiumSkyblock.getInstance().getConfiguration().voidTeleport) {
```
Comment (MertUnverdi) [inline src/main/java/com/iridium/iridiumskyblock/listeners/PlayerMoveListener.java:53]: U can check with this
```suggestion
        if (event.getTo().getY() < 0 && IridiumSkyblock.getInstance().getConfiguration().voidTeleport && IridiumSkyblockAPI.getInstance().isIslandWorld(player.getWorld()) {
```

Comment (xReveres) [reply src/main/java/com/iridium/iridiumskyblock/listeners/PlayerMoveListener.java:53]: @PeachesMLG @dlsf  did you think this is good? or better is to get world from event location?

Comment (MertUnverdi) [reply src/main/java/com/iridium/iridiumskyblock/listeners/PlayerMoveListener.java]: lol i forgot a bracket

**Review by dlsf — APPROVED** [review]

**Review by MertUnverdi — APPROVED** [review]

**Pull request merged**